.github/workflows/push-dist.yml

```diff
@@ -0,0 +1,25 @@
+# Because this library needs to be built,
+# we can't easily point package.json files at the git repo for easy cross-repo testing.
+#
+# This workflow brings back that capability by placing the compiled assets on a "dist" branch
+# (configurable via the "branch" option below)
+name: Push dist
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  push-dist:
+    name: Push dist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: NullVoxPopuli/action-setup-pnpm@v2
```

Comment:
```suggestion
      - uses: wyvox/action-setup-pnpm@v3
```

Reply:
Fair enough. Suggest you [update this in the addon blueprint](https://github.com/search?q=repo%3Aembroider-build%2Faddon-blueprint%20NullVoxPopuli%2Faction-setup-pnpm&type=code)

Reply:
yes!